src/main/java/core/basesyntax/service/StorageLiner.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.StorageDao;
+
+public interface StorageLiner {
+    String getLines(StorageDao storageDao);
+}

```

Comment:
```suggestion
public interface ReportService {
    String createReport();
}

```
and let's pass `StorageDao` through constructor